src/punctuation.js

```diff
@@ -29,6 +20,14 @@ export default async function () {
 		return rules;
 	}
 
+	function getRules(recordType) {
+		if (recordType === 'z') {
+			return require('../src/auth-punctuation.json');
+		}
+
+		return require('../src/bib-punctuation.json');
```

Comment:
Didn't notice this earlier. We don't use `require` anymore (It's Node.js -specific module loading method) but std ES `import` instead. That, however, does not support non-JS files, so you need to wrap those files in Javascript and import using `import` as usual:
```js
export default const {
...
}
```